### DISCUSSION TITLE
About website domain

### DISCUSSION THREAD

**Discussion by ZLY201** [js-sdsl/js-sdsl#64] [Ideas]
I want to buy a domain from google.

I think the `js-sdsl.org` is a good choice but I don't like the `-` character and `jsSdsl.org` isn't beautiful.

Do you have any good idea? @noname0310

**Comment by noname0310** [comment]
I think js-sdsl.org would be good.

**Comment by ZLY201** [comment]
Do you have a opencollective account? @noname0310 

I want to create a new collective but it must have two people to do managment.

https://opencollective.com/

**Reply by noname0310** [reply]
I just signed up

The email in my account is hjnam2014@gmail.com

**Reply by ZLY201** [reply]
I invited you, please check.

**Comment by ZLY201** [comment]
Good news, [js-sdsl.org](https://js-sdsl.org) is enabled!
